__tests__/babyAnimal.test.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+require('@code-fellows/supergoose');
+
+const BabyAnimal = require('../models/babyAnimal-collection.js');
```

Comment:
This isn't linking right, it needs to be: `'../src/models/babyAnimal-collection'`